modules/util/portal-tools-rest-builder/src/main/resources/com/liferay/portal/tools/rest/builder/dependencies/dto.ftl

```diff
@@ -281,6 +290,27 @@ public class ${schemaName} <#if dtoParentClassName?has_content>extends ${dtoPare
 		return Objects.equals(toString(), ${schemaVarName}.toString());
 	}
 
+	<#if generateGetValues?has_content>
+		<#assign firstPropertyName = generateGetValues[0]>
+			public Object getValue(String propertyName) {
+				if (Objects.equals(propertyName, "${firstPropertyName}")) {
+					return ${firstPropertyName};
+		<#assign reducedProperties = generateGetValues[0..generateGetValues?size - 1]>
+		<#list reducedProperties as propertyName>
```

Comment:
This is very complex. I think we should be able to include all the cases in a for loop for better understanding

Reply:
I think you can use `<#sep> [....] </#sep>` for this purpose.